app/waf/src/controllers/endpointpolicy.controller.ts

```diff
@@ -37,9 +36,6 @@ export class EndpointpolicyController {
   async create(
     @requestBody() endpointpolicy: Partial<Endpointpolicy>,
   ): Promise<Endpointpolicy> {
-    if (!endpointpolicy.id) {
-      endpointpolicy.id = uuid();
-    }
     return await this.endpointpolicyRepository.create(endpointpolicy);
```

Comment:
If no id input from API, the uuid is not generated here, right? Did you test that?

Reply:
will check again.

Reply:
pick it back.